### PR TITLE
f-checkout@0.158.0 - Migrate to global cookie handler in `adddressServices`

### PR DESCRIPTION
**Pull request by oliversweb**
### changed
- Removed the direct get postcode cookie call from `addressService.js` and now pass value in.

### CODE REVIEW

File: packages/components/organisms/f-checkout/src/services/addressService.js [hunk 1]
```diff
@@ -126,9 +116,8 @@ function getAddressCoordsFromLocalStorage () {
     return null;
 }
 
-
 /**
- * Checkes whether the address values in local storage match what's is in the form.
+ * Checks whether the address values in local storage match what's is in the form.
```
Comment (simonsandersje) [inline packages/components/organisms/f-checkout/src/services/addressService.js:120]: Going for your "Leave the code tidy" Boy Scout badge?

**Review by simonsandersje — DISMISSED** [review]

File: packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js [hunk 2]
```diff
@@ -603,6 +604,7 @@ describe('CheckoutModule', () => {
         describe('getAddress ::', () => {
             it(`should get the address details from the backend and call ${UPDATE_FULFILMENT_ADDRESS} mutation.`, async () => {
                 // Arrange
+                payload = { ...payload, currentPostcode: 'JZ11AA' };
```
Comment (BanksySan) [inline packages/components/organisms/f-checkout/src/store/tests/checkout.module.test.js]: Could we give this magic string a name?  If it's just any valid postcode then putting it in a const called something like `DUMMY_POSTCODE` or `VALID_POSTCODE`.

**Review by BanksySan — DISMISSED** [review]

File: packages/components/organisms/f-checkout/src/services/addressService.js (hunk 1 above)
Comment (oliversweb) [reply packages/components/organisms/f-checkout/src/services/addressService.js:120]: lol, always `sir`

File: packages/components/organisms/f-checkout/src/services/addressService.js [hunk 3]
```diff
@@ -138,19 +127,28 @@ function doesAddressInStorageAndFormMatch (storedAddress, formAddress) {
         && storedAddress.Line2 === formAddress.line2 && storedAddress.City === formAddress.locality;
 }
 
-export default {
-    doesAddressInStorageAndFormMatch,
-    getClosestAddress (addresses, tenant) {
```
Comment (kevinrodrigues) [inline packages/components/organisms/f-checkout/src/services/addressService.js:143]: ⚡

**Review by kevinrodrigues — DISMISSED** [review]

**Review by simonsandersje — DISMISSED** [review]

**Review by simonsandersje — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**